www_src/blocks/all.jsx

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Just a utility to get the right block, since this is used a lot. Replaces `generator.js` below